index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS practice</title>
+    <link rel="stylesheet" type="text/css" href="app.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <nav>
+      <ul class="navList">
+        <li class="inscription"><a href="#" class="ref">About</a></li>
+        <li class="inscriptionDeviate"><a href="https://youtu.be/xdYYN-4ttDg" class="ref" target="_blank">HOBBY</a></li>
```

Comment:
Avoid writing uppercase text in HTML. Visual representing must be implemented in CSS.
For example: when you change content your visual style must be saved and not depend on the markup.